WriteVHDLSyntax.py

```diff
@@ -747,6 +800,14 @@ def writeMemoryRHSPorts_interface(mtypeB, memInfo, memDict):
           string_output_mems += "    "+mem+"_enb_nent        : out t_"+mtypeB+"_1b;\n"
           string_output_mems += "    "+mem+"_V_addr_nent        : out t_"+mtypeB+"_NENTADDR"+disk+";\n"
           string_output_mems += "    "+mem+"_AV_dout_nent       : out t_"+mtypeB+"_NENT;\n"
+      elif "AS" in mtypeB and split: #AS/TPAR at interface need to go through merging module
+          string_output_mems += "    "+mem+"_stream_V_dout : out std_logic_vector(36 downto 0);\n"
```

Comment:
@jasonfan393 Can we avoid the hard-coded `36`?
```suggestion
          string_output_mems += "    "+mem+"_stream_V_dout : out std_logic_vector(" + str(memMod.bxbitwidth + memMod.bitwidth - 1) + " downto 0);\n"
```

Or maybe the type should be defined in memUtil_pkg.vhd?